scapy/arch/bpf/core.py

```diff
@@ -70,8 +70,8 @@ def get_if_raw_hwaddr(ifname):
         raise Scapy_Exception("Failed to execute ifconfig: (%s)" % msg)
 
     # Get MAC addresses
-    addresses = [l for l in fd.readlines() if l.find("ether") >= 0 or
-                 l.find("lladdr") >= 0 or
+    addresses = [l for l in fd.readlines() if l.find("ether") >= 0 or  # noqa: W504, E501
```

Comment:
Could you move the `if` to a next line and remove at least the `E501` new `noqa`?

Reply:
rewritten with `in` instead of `find()` to save space.